src/SFA.DAS.Campaign.Web/Controllers/EmployerController.cs

```diff
@@ -1,23 +1,106 @@
+        private const string calculationPanel1Slug = "are-you-ready-to-get-going";
+        private const string calculationPanel2Slug = "future-proof-your-business";
+
+        public EmployerController(IOptions<CampaignConfiguration> configuration, IMediator mediator, IStandardsRepository repository)
         {
             _configuration = configuration.Value;
+            _mediator = mediator;
+            _repository = repository;
         }
-        
+
         [Route("/employers/find-apprenticeship-training")]
         [Route("/employer/find-apprenticeship-training")]
         public IActionResult FindApprenticeshipTraining()
         {
             return RedirectPermanent(_configuration.FatBaseUrl);
         }
+
+        [Route("/employers/calculate-your-apprenticeship-funding")]
+        public async Task<IActionResult> CalculateApprenticeshipFunding([FromQuery] bool preview = true)
+        {
+            var standards = _repository.GetStandards();
+            var staticContent = _mediator.GetModelForStaticContent();
+            var panel1 = _mediator.Send(new GetPanelQuery() { Slug = calculationPanel1Slug, Preview = preview });
+            var panel2 = _mediator.Send(new GetPanelQuery() { Slug = calculationPanel2Slug, Preview = preview });
+
+            await Task.WhenAll(staticContent, panel1, panel2, standards);
+
+            return View(new ApprenticeshipFundingViewModel
+            {
+                Standards = standards.Result,
+                Menu = staticContent.Result.Menu,
+                BannerModels = staticContent.Result.BannerModels,
+                Panel1 = panel1.Result.Panel,
+                Panel2 = panel2.Result.Panel,
+                Submitted = false
+            });
+        }
+
+        [HttpPost("/employers/calculate-your-apprenticeship-funding")]
+        public async Task<IActionResult> CalculateApprenticeshipFunding(ApprenticeshipFundingViewModel model, [FromQuery] bool preview = true)
+        {
+            var standardResult = _repository.GetStandard(model.StandardUid);
+            var staticContent = _mediator.GetModelForStaticContent();
+            var panel1 = _mediator.Send(new GetPanelQuery() { Slug = calculationPanel1Slug, Preview = preview });
+            var panel2 = _mediator.Send(new GetPanelQuery() { Slug = calculationPanel2Slug, Preview = preview });
+            await Task.WhenAll(staticContent, panel1, panel2, standardResult);
+
+            var standard = new Standard
+            {
+                Title = standardResult.Result.Title,
+                StandardUId = standardResult.Result.StandardUId,
+                LarsCode = standardResult.Result.LarsCode,
+                Level = standardResult.Result.Level,
+                MaxFunding = standardResult.Result.MaxFundingAvailable,
+                Duration = standardResult.Result.TimeToComplete
+            };
+
+            CalculationOutputValues result = new CalculationInputValues()
```

Comment:
The Calculation Input / Output I think can also be changed into a mediator handler to encapsulate the logic and take a lot of the scaffolding out of the controller and reduce the number of new POCO we have along with model conversion